packages/wrangler/src/versions/secrets/index.ts

```diff
@@ -209,6 +209,7 @@ export async function copyWorkerVersionWithNewSecrets({
 			"workers/message": versionMessage,
 			"workers/tag": versionTag,
 		},
+		keep_assets: true,
```

Comment:
Oh I see! I really don't like this, there should be a versions secrets API endpoint that Wrangler can hit

Reply:
Totally agree.